source/developer/dev-setup-ubuntu-1604.rst

```diff
@@ -39,9 +39,13 @@ Set up your development environment for building, running, and testing Mattermos
 
        ``sudo tar -C /usr/local -xzf go1.8.linux-amd64.tar.gz``
 
-    c. Modify permissions on ``/usr/local/go``. Replace *{user}* and *{group}* with the user and group that you are logged in with.
+    c. Ensure the binary file of golang have the right permission, by running:
```

Comment:
If you tried that and that was the result, then yes, it could be completely removed. I have not tested it myself but normally when you extract a tarball it keeps the permissions, I can't speak for all *nix, but at the very least that's how it is for CentOS 5~7 and apparently for Ubuntu. So I will remove c and d.